exchanges.yml

```diff
@@ -217,49 +217,81 @@ components:
                     type: string
                   template:
                       type: string
-            workflowId:
+            id:
```

Comment:
`id` can be optional within the `WorkflowConfig` object when creating a workflow, indicating either a preferred `id` (which some servers may honor, others may reject).